modules/crs-generate-roxctl.adoc

```diff
@@ -0,0 +1,119 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_helm/install-helm-customization.adoc
+//
+// You must declare the `topic-helm` or `topic-operator` attribute when using this module.
+:_mod-docs-content-type: PROCEDURE
+[id="crs-generate-roxctl_{context}"]
+= Generating a CRS by using the roxctl CLI
+
+ifeval::["{context}" == "init-bundle-cloud-other"]
+:other:
+endif::[]
+
+ifeval::["{context}" == init-bundle-cloud-ocp"]
+:openshift:
+endif::[]
+
+ifeval::["{context}" == "init-bundle-cloud-ocp-generate"]
+:cloud-ocp-generate:
+endif::[]
+
+ifeval::["{context}" == "init-bundle-cloud-other-generate"]
+:cloud-other-generate:
+endif::[]
+
+You can create a cluster registration secret by using the `roxctl` CLI.
+
+[NOTE]
+====
+You must have the `Admin` user role to create a CRS.
+====
+
+.Prerequisites
+
+* You have configured the `ROX_API_TOKEN` and the `ROX_CENTRAL_ADDRESS` environment variables:
+
+.. Set the `ROX_API_TOKEN` by running the following command:
++
+[source,terminal]
+----
+$ export ROX_API_TOKEN=<api_token>
+----
+
+.. Set the `ROX_CENTRAL_ADDRESS` environment variable by running the following command:
++
+[source,terminal]
+----
+$ export ROX_CENTRAL_ADDRESS=<address>:<port_number>
+----
+
+ifdef::cloud-ocp-generate,cloud-other-generate[]
+[IMPORTANT]
+====
+In {product-title-managed-short}, when using `roxctl` commands that require the Central address, use the *Central instance address* as displayed in the *Instance Details* section of the {cloud-console}. For example, use `acs-ABCD12345.acs.rhcloud.com` instead of `acs-data-ABCD12345.acs.rhcloud.com`.
+====
+endif::cloud-ocp-generate,cloud-other-generate[]
+
+.Procedure
+
+* To generate a CRS, run the following command:
++
+[source,terminal]
+----
+$ roxctl central crs generate <crs_file_name> --output <string> <1>
+----
+<1> Enter a file name or use `-` for standard output.
++
+[IMPORTANT]
+====
+Ensure that you store this file securely because it contains secrets.
+You can use the same file to set up multiple secured clusters. You cannot retrieve a previously-generated CRS.
+====
+
+.Sample output
+
+[source,terminal]
+----
+INFO:	Successfully generated new CRS
+INFO:
+INFO:	  Name:       test-crs
+INFO:	  Created at: 2025-02-26T19:07:21Z
+INFO:	  Expires at: 2026-02-26T19:07:00Z
+INFO:	  Created By: sample-token
+INFO:	  ID:         9214a63f-7e0e-485a-baae-0757b0860ac9
+# This is a StackRox Cluster Registration Secret (CRS).
+# It is used for setting up StackRox secured clusters.
+# NOTE: This file contains secret data that allows connecting new secured clusters to central,
+# and needs to be handled and stored accordingly.
+apiVersion: v1
+data:
+  crs: EXAMPLEZXlKMlpYSnphVzl1SWpveExDSkRRWE1pT2xzaUxTMHRMUzFDUlVkSlRpQkRSVkpVU1VaSlEwREXAMPLE=
```

Comment:
Added `example` to try to stop ITSec from alerting on this example, but it's not working.

Reply:
do they have any suggested sample values for these? seems like the kind of thing that should exist somewhere :sweat_smile:

Reply:
That is a good question...I'll see if I can find out!

Reply:
I think it did stop alerting on my example, or at least I haven't gotten any more "You have breached security!" emails.